src/index.js

```diff
@@ -1,14 +1,17 @@
 import componentsPlugin from './components'
 import directivesPlugin from './directives'
-import { installFactory, vueUse } from './utils/plugins'
+import { installFactory } from './utils/plugins'
 import { setConfig } from './utils/config'
 
+// Named export of all compoents and component group plugins
+export * from './components/index.esm'
+
+// Named export of all directives and directive group plugins
+export * from './directives/index.esm'
+
 const BootstrapVue = {
```

Comment:
Do we need to keep this object?

Reply:
This exports all the `VB<directive>Plugin` and `VB<directive>` exports... or are you referring to the `BootstrapVue` object?

Reply:
Yes `BootstrapVue` object in unsed

Reply:
its used by the `es/` builds, and by `browser.js`

need to keep it for backwards compatibility